unified-codes/apps/web/src/actions/ExplorerActions.ts

```diff
@@ -1,247 +1,252 @@
 import { Action } from 'redux';
-
 import { EEntityField, IEntity } from '@unified-codes/data';
+import { IExplorerParameters } from '../types';
 
 export interface IExplorerSearchBarUpdateInputAction extends Action<string> {
-    input: string;
+  input: string;
 }
 
 export interface IExplorerSearchBarUpdateFilterByAction extends Action<string> {
-    filterBy: EEntityField;
+  filterBy: EEntityField;
 }
 
-
 export interface IExplorerTableUpdateRowsPerPageAction extends Action<string> {
-    rowsPerPage: number;
+  rowsPerPage: number;
 }
 
 export interface IExplorerTableUpdatePageAction extends Action<string> {
-    page: number;
+  page: number;
 }
 
 export interface IExplorerTableUpdateOrderByAction extends Action<string> {
-    orderBy: EEntityField;
+  orderBy: EEntityField;
 }
 
 export interface IExplorerTableUpdateOrderDescAction extends Action<string> {
-    orderDesc: boolean;
+  orderDesc: boolean;
 }
 
 export interface IExplorerTableUpdateFilterByAction extends Action<string> {
-    filterBy: string;
+  filterBy: string;
 }
 
-export interface IExplorerTableFetchEntitiesAction extends Action<string> { };
+export interface IExplorerTableFetchEntitiesAction extends Action<string> {
+  parameters?: IExplorerParameters;
```

Comment:
I think this is unnecessary? I think either the saga should use the selectors, or we should pass them via the action.  Since the store is the source of truth, I think it doesn't really matter which way we go, but I think we should choose one?

Reply:
Oops, just read the description: 

```
this allows updating of params for the search, without needing to chain dispatches or wait for the store to update before fetching.
```

I think we want to maintain the "state machine" logic. So I think chaining dispatches is probably the best way to do it... otherwise it can end up introducing complexities and tricky bugs. I don't think there;s a performance need to do this, so I would suggest keeping it simple? Thoughts?